tools/terraform_sync_tool/modules/terraform-sync-tool/main.tf

```diff
@@ -0,0 +1,96 @@
+locals {
```

Comment:
Rename the module to something else. You currently have the module directory named "terraform-sync-tool" which is misleading because the terraform is not here to set up the sync tool. The terraform serves only as example terraform.

Reply:
Renamed module to bq-setup - https://github.com/candicehou07/bigquery-utils/commit/2a354f0fbe0428a9daabcb476c79f48938ab9c07